doc/particles.md

```diff
@@ -96,110 +99,110 @@ game.add(
 )
 ```
 
-You can find more examples of using different built-int particles in various combinations [here](https://github.com/flame-engine/flame/tree/master/doc/examples/particles/lib/main.dart).
+You can find more examples of using different built-in particles in various combinations [here](https://github.com/flame-engine/flame/blob/main/examples/lib/stories/utils/particles.dart).
+
 
 ## Lifecycle
 
-Behavior common to all `Particle`s is that all of them accept a `lifespan` parameter. This value is used to make the `ParticleComponent` self-remove, once its internal `Particle` has reached the end of its life. Time within the `Particle` itself is tracked using the Flame `Timer`. It could be configured with a `double`, representing seconds (with microsecond precision) by passing it into the corresponding `Particle` constructor.
+Behavior common to all `Particle`s is that all of them accept a `lifespan` parameter. This value is used to make the `ParticleComponent` self-remove, once its internal `Particle` has reached the end of its life. Time within the `Particle` itself is tracked using the Flame `Timer` class. It can be configured with a `double`, represented in seconds (with microsecond precision) by passing it into the corresponding `Particle` constructor.
```

Comment:
```suggestion
A behavior common to all `Particle`s is that all of them accept a `lifespan` argument. This value is used to make the `ParticleComponent` remove itself once its internal `Particle` has reached the end of its life. Time within the `Particle` itself is tracked using the Flame `Timer` class. It can be configured with a `double`, represented in seconds (with microsecond precision) by passing it into the corresponding `Particle` constructor.
```